src/app/components/grid/index.html

```diff
@@ -80,7 +80,7 @@
       propertyName="settingsKey"
       isOptional="true"
     >
-    Specifies a unique key for the UI Config Service to retrieve stored settings from a database. You must prefix the key with the name of your SPA. The UI Config Service applies sticky settings to grids and saves configuration settings for users. This property accepts <sky-code>string</sky-code> values. This property is for internal Blackbaud use only. For Blackbaud developers, the UI Config Service website describes how to <a href="https://docs.blackbaud.com/ui-config-service-docs/learn/tutorials/save-grid-configuration">apply sticky settings to grids</a>.
+    Specifies a unique key for the UI Config Service to retrieve stored settings from a database. You must prefix the key with the name of your SPA. The UI Config Service saves configuration settings for users and returns <sky-code>selectedColumnIds</sky-code> to preserve their preferred column order and the sorting of columns. This property accepts <sky-code>string</sky-code> values. This property is for internal Blackbaud use only. For Blackbaud developers, the UI Config Service website describes how to <a href="https://docs.blackbaud.com/ui-config-service-docs/learn/integrations/skyux-components/grids">apply sticky settings to grids</a>.
```

Comment:
> You must prefix the key with the name of your SPA.

Might check with @Blackbaud-TerryHelems but I believe this is not necessary here, as our config service does that for you.

Reply:
For example, a consumer would declare a key like so:
```
[settingsKey]="'event-list'"
```
NOT like this:
```
[settingsKey]="'skyux-spa-events-event-list'"
```

Reply:
I confirmed with Terry and removed the second sentence.